x-pack/plugin/rollup/src/test/java/org/elasticsearch/xpack/downsample/TransportDownsampleActionTests.java

```diff
@@ -98,6 +100,7 @@ private static void assertTargetSettings(final IndexMetadata indexMetadata, fina
             indexMetadata.getSettings().get(IndexMetadata.INDEX_BLOCKS_WRITE_SETTING.getKey()),
             settings.get(IndexMetadata.INDEX_BLOCKS_WRITE_SETTING.getKey())
         );
+        assertEquals(indexMetadata.getSettings().get(LifecycleSettings.LIFECYCLE_NAME), settings.get(LifecycleSettings.LIFECYCLE_NAME));
```

Comment:
Maybe we should add this setting to `source` variable at line 25? Then we need to assert it isn't here? So check for null?

Reply:
We can't because it is "private"